src/web/components/Caption.tsx

```diff
@@ -84,7 +86,7 @@ export const Caption: React.FC<{
     };
 
     const shouldLimitWidth =
-        role && (role === 'showcase' || role === 'supporting');
+        !isMainMedia && role && (role === 'showcase' || role === 'supporting');
```

Comment:
This is a fix for the caption being styled as though in body when in fact it's main media. It's not actually related to grid but got boyscouted into this PR

Reply:
+1

Reply:
I guess `role` here is extra as `(role === 'showcase' || role === 'supporting')` can never be true so could drop it?